lua/trevj.lua

```diff
@@ -28,8 +28,44 @@ local make_c_containers = function()
   }
 end
 
+local make_javascript_typescript_containers = function()
+  local javascript = {
+    array = make_default_opts(),
+    object = make_default_opts(),
+    arguments = make_default_opts(),
+    namedimports = make_default_opts(),
+    object_pattern = make_default_opts(),
+    formal_parameters = make_default_opts(),
+  }
+
+  local jsx = {
+    jsx_element = make_no_final_sep_opts(),
+    jsx_opening_element = {
+      final_separator = false,
+      final_end_line = true,
+      skip = { identifier = true },
+    },
+  }
+  local typescript_only = {
+    type_parameters = make_default_opts(),
+    type_arguments = {
+      final_separator = false,
+      final_end_line = true,
+    },
```

Comment:
```suggestion
    type_arguments = make_no_final_sep_opts(),
```
[nit]

Reply:
Good idea, I missed it :smile: I have applied your suggestion by amending the commit to avoid unnecessary style-related commits in the history